src/owlwatch/schema/model.py

```diff
@@ -180,10 +180,20 @@ def from_json(cls, ip_json):
         # Get index pattern fields
         fields_json = json.loads(ip_json['value']['fields'])
         for json_field in fields_json:
+            if 'scripted' in json_field and json_field['scripted']:
+                # Don't check Kibana generated scripted fields
+                continue
+            elif 'is_' in json_field['name']:
```

Comment:
This should be done via Schema.__excluded_props . Even better, using another exclusion list, passed as parameter to Schema class at some point, so the user would be able to exclude custom lists of fields.